master/reference/calicoctl/resources/policy.md

```diff
@@ -44,20 +44,26 @@ spec:
 | name | The name of the policy. |         | string |
 
 
-#### Spec 
+#### Spec
+
+| Field      | Description                                                                                                                                                         | Accepted Values | Schema                | Default |
+|------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------+-----------------------+---------|
+| order      | (Optional) Indicates priority of this policy, with lower order taking precedence.  No value indicates highest order (lowest precedence)                             |                 | float                 |         |
+| selector   | Selects the endpoints to which this policy applies.                                                                                                                 |                 | [selector](#selector) | all()   |
+| ingress    | Ordered list of ingress rules applied by policy.                                                                                                                    |                 | List of [Rule](#rule) |         |
+| egress     | Ordered list of egress rules applied by this policy.                                                                                                                |                 | List of [Rule](#rule) |         |
+| doNotTrack | Indicates that the rules in this policy should be applied before any data plane connection tracking, and that packets allowed by these rules should not be tracked. | true, false     | boolean               | false   |
 
-| Field    | Description                 | Accepted Values   | Schema | Default    |
-|----------|-----------------------------|-------------------|--------|------------|
-| order    | (Optional) Indicates priority of this policy, with lower order taking precedence.  No value indicates highest order (lowest precedence) | | float |  |
-| selector | Selects the endpoints to which this policy applies. | | [selector](#selector)| all() |
-| ingress  | Ordered list of ingress rules applied by policy. | | List of [Rule](#rule)  | |
-| egress   | Ordered list of egress rules applied by this policy. | | List of [Rule](#rule)  | |
+The `doNotTrack` field is meaningful for [host
```

Comment:
Maybe be even stronger here: untracked policies don't apply to workload endpoints at all in the current implementation.